db/batch/batch_impl.go

```diff
@@ -270,7 +270,7 @@ func (cb *cachedBatch) Delete(namespace string, key []byte, errorFormat string,
 	h := cb.hash(namespace, key)
 	cb.touchKey(h)
 	cb.currentCache().Evict(&h)
-	cb.kvStoreBatch.batch(Delete, namespace, key, nil, errorFormat, errorArgs)
+	cb.kvStoreBatch.batch(Delete, namespace, key, nil, errorFormat, errorArgs...)
```

Comment:
this is prob not the complete fix, need to also change `errorArgs` to `[]interface{}`, and use `errorArgs...` when it is used in `WriteBatch`
see https://github.com/iotexproject/iotex-core/commit/010f0d08aa94a35a4aec943da5034167ad79be8b (also not complete, just demo the fix on `baseKVStoreBatch`)